arm-documentdb/2015-04-08/swagger/documentdb.json

```diff
@@ -336,24 +336,18 @@
           }
         ],
         "responses": {
-          "200": {
-            "description": "The access key was regenerated successfully.",
-            "schema": {
-              "$ref": "#/definitions/DatabaseAccountListKeysResult"
-            }
+          "202": {
+            "description": "Accepted. The regenerate key operation will complete asynchronously."
           }
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.DocumentDB/databaseAccountNames/{accountName}": {
+    "/providers/Microsoft.DocumentDB/databaseAccountNames/{accountName}": {
```

Comment:
Our database account names are unique across the entire service and so it didn't make sense to force the users to enter a subscriptionId.